core/src/main/scala/io/finch/RequestReaders.scala

```diff
@@ -135,6 +136,18 @@ trait RequestReaders {
   val binaryBody: RequestReader[Array[Byte]] = binaryBodyOption.failIfNone
 
   /**
+   * A [[RequestReader]] that reads a required chunked streaming binary body, interpreted as a `AsyncStream[Buf]`.
+   */
+  def asyncBody(chunkSize: Int): RequestReader[AsyncStream[Buf]] = {
```

Comment:
Int.MaxValue is 2GB. Since it's not always guaranteed that the server has enough room for allocating 2GB buffer I added the parameter `chunkSize`. Not sounds reasonable?